go-controller/pkg/node/gateway_udn.go

```diff
@@ -57,7 +58,11 @@ func (udng *UserDefinedNetworkGateway) AddNetwork() error {
 	}
 	vrfDeviceName := util.GetVRFDeviceNameForUDN(udng.networkID)
 	vrfTableId := util.CalculateRouteTableID(mplink.Attrs().Index)
```

Comment:
i just realised that when you create the mp above on ln 55, it adds the mp address to it but when you enslave mp to the device of type VRF, then for IPv6 address, theyre going to be removed (except link local)

Reply:
can be done as follow up fix

Reply:
ack and I know you already fixed this via https://github.com/ovn-org/ovn-kubernetes/pull/4626 ? so can I resolve this comment?